engine/src/main/java/com/datatorrent/stram/cli/ApexCli.java

```diff
@@ -674,7 +674,7 @@ void printUsage(String cmd)
         "Get the configuration parameter"));
     globalCommands.put("get-app-package-info", new CommandSpec(new GetAppPackageInfoCommand(),
         new Arg[]{new FileArg("app-package-file")},
-        null,
+        new Arg[]{new VarArg("--with-description")},
```

Comment:
Please change to `-with-description` to be consistent. I believe apex uses one dash for options.